storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1245,6 +1248,11 @@ struct my_option xb_client_options[] = {
      &opt_generate_transition_key, &opt_generate_transition_key, 0, GET_BOOL,
      NO_ARG, 0, 0, 0, 0, 0, 0},
 
+     {"keyring-file-data", OPT_KEYRING_FILE_DATA,
```

Comment:
didn't we have this option name already? :thinking: how did we pass keyring_file_data before?

Reply:
@satya-bodapati we did have it before as part of the keyring plugin, if the keyring_plugin.so is read parsing that parameter will result on opt_keyring_file_data been available, however, to keep the same parameters for keyring component (when the plugin is not read) we need to make pxb understand that command line parameter.